src/momento/typing.py

```diff
@@ -0,0 +1,14 @@
+from typing import Union
+
+# Scalar Types
+TScalarKey = Union[str, bytes]
+TScalarValue = Union[str, bytes]
+
+# Collections
+TCollectionKey = str
```

Comment:
The term used by collections is name. TCollectionKey and TScalarKey are different things.

```suggestion
TCollectionName = str
```